src/adagrams.js

```diff
@@ -1,7 +1,97 @@
+const _ = require('lodash');
+
 const Adagrams = {
   drawLetters() {
-    // Implement this method for wave 1
+    const allLetters = ["A", "A", "A", "A", "A", "A", "A", "A", "A", "B", "B", "C", "C", "D", "D", "D", "D", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "F", "F", "G", "G", "G", "H", "H", "I", "I", "I", "I", "I", "I", "I", "I", "I", "J", "K", "L", "L", "L", "L", "M", "M", "N", "N", "N", "N", "N", "N", "O", "O", "O", "O", "O", "O", "O", "O", "P", "P", "Q", "R", "R", "R", "R", "R", "R", "S", "S", "S", "S", "T", "T", "T", "T", "T", "T", "U", "U", "U", "U", "V", "V", "W", "W", "X", "Y", "Y", "Z"];
+
+    //use lodash .sampleSize method to get a random sample of 10 letters 
+    const lettersInHand = _.sampleSize(allLetters, 10);
+
+    return lettersInHand;
+
+  },
+
+  usesAvailableLetters(input, lettersInHand) {
+    input = input.toUpperCase();
+
+    let lettersObject = {};
```

Comment:
turns out that `lettersObject` can be a `const` because you never re-assign it, even if you modify it